packages/web3-eth-accounts/src/types.ts

```diff
@@ -47,49 +47,6 @@ export type SignFunction = (data: string, privateKey: string) => SignResult;
 
 // https://github.com/ethereum/wiki/wiki/Web3-Secret-Storage-Definition
 
-export type Cipher = 'aes-128-ctr' | 'aes-128-cbc' | 'aes-256-cbc';
```

Comment:
looks good , however for this types moving to types package , mention in package and also root level readme files along with PR#.

Reply:
done